note/note_cosigv1.go

```diff
@@ -107,6 +107,20 @@ func NewVerifierForCosignatureV1(vkey string) (note.Verifier, error) {
 	return v, nil
 }
 
+// CoSigV1Timestamp extracts the embedded timestamp from a CoSigV1 signature.
+func CoSigV1Timestamp(s note.Signature) (time.Time, error) {
+	r, err := base64.StdEncoding.DecodeString(s.Base64)
+	if err != nil {
+		return time.UnixMilli(0), errMalformed
+	}
+	if len(r) != 4+8+ed25519.SignatureSize {
```

Comment:
Could the `4` and `8` be constants to tell what they are?

Reply:
I was following the style elsewhere in this file, but I've added two consts.